ptsrc/parser/scan.ssl

```diff
@@ -161,8 +176,36 @@ Scan :
                 .pLeftBracket
             | ']':
                 .pRightBracket
+            | '{':
+                .pLeftBrace
+            | '}':
+                .pRightBrace
             | ',':
                 .pComma
+            | '?':
+                .pQuestionMark
+            | '/':
+                .pSlash
+            | '%':
+                .pPercent
+            | '&':
+                [
+                    | '&':
+                        .pDoubleAmpersand
+                    | *:
+                        .pAmpersand
+                ]
+            | '|':
+                [
+                    | '|':
+                        .pBarBar
+                    | *:
+                        .pBar
```

Comment:
I think this needs to be changed to .pOrBar

Reply:
It is just a naming convention I can change it